oslat-client

```diff
@@ -91,7 +102,14 @@ echo "about to run: ${cmd}"
 CMD_OUTPUT=$(${cmd})
 echo -e "${CMD_OUTPUT}"
 WORKLOAD_CPUS=$(echo -e "${CMD_OUTPUT}" | grep "final cpus:" | awk '{ print $3 }')
-echo "WORLOAD_CPUS: ${WORKLOAD_CPUS}"
+echo "WORKLOAD_CPUS: ${WORKLOAD_CPUS}"
+
+if [[ -n "${cpu_list}" ]]; then
+    echo "Overriding WORKLOAD_CPUS: $WORKLOAD_CPUS w/ cpu-list: ${cpu_list}"
+    WORKLOAD_CPUS=${cpu_list}
+    echo "WORKLOAD_CPUS = $WORKLOAD_CPUS"
+fi
```

Comment:
So I'm a little curious what the use case is for this option?

Reply:
Right...but are we ever actually doing that?

Reply:
No...
We always automatically guessed/detected the workload cpus and housekeeping cpus.  We currently do NOT allow the user to select a custom range of cpus. This is the [original code](https://github.com/perftool-incubator/bench-oslat/blob/master/oslat-client#L133):
`
cmd="taskset -c ${HK_CPUS},${WORKLOAD_CPUS} oslat --duration ${duration} ${rtprio_opt} --cpu-list ${WORKLOAD_CPUS} --cpu-main-thread ${cpu_main_thread} ${trace_args}"
`

We used to measure latency with the workload cpus. Then, more recently, we added the cpu-main-thread [param](https://github.com/perftool-incubator/bench-oslat/blob/master/oslat-client#L107C1-L108C60)
 by automatically selecting the first cpu from the workload cpus range:
`
# get the first CPU from HK_CPUS to use as the oslat main thread
cpu_main_thread=$(echo ${HK_CPUS} | awk -F, '{ print $1 }')
`
Now the proposal in this PR is to let the user specify in the JSON run file what is the cpu-list for measuring latency (--cpu-list param), and what is the houskeeping cpu range (--house-keeping param). Default behavior, if not specified, continue to be: automatically detect the workload/hk cpus.

Future PR to validate if the user range is valid.... For now we assume that if cpus are specified, user knows what he is doing.